puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/builders/gemset_create.yaml

```diff
@@ -0,0 +1,4 @@
+- builder:
+    name: gemset_create
+    builders:
+      - shell: !include-raw scripts/gemset_create.sh
```

Comment:
> http://ci.theforeman.org/job/test_katello_core/database=postgresql,ruby=1.9.3,slave=fast/16669/console

This looks like it's using Ruby 1.8.7, given the number of errors.  I guess it didn't inherit the rvm environment.